lib/src/components/NxCollapsibleItems/NxCollapsibleItems.scss

```diff
@@ -92,11 +92,16 @@ $-form-child-left-padding: calc(#{$-trigger-side-padding} + #{$-triangle-box-wid
   }
 }
 
+%collapsible-items-focus-style {
+  outline: 2px solid var(--nx-color-interactive-border-focus);
+  outline-offset: -2px;
+}
+
 .nx-collapsible-items__trigger {
   @include nx-container-helpers.container-horizontal;
   align-items: center;
   background: none;
-  border: 1px solid transparent;
+  border: 0;
```

Comment:
I think if we should leave this as it was. Kaleido shows that the height of the title is still `32px`, whereas with this change, the height of the title is now `30px`, but the children are still `32px`. Reverting this change would fix the screenshots' "height" differences and still be in line with Kaleido.

Reply:
> I think if we should leave this as it was. Kaleido shows that the height of the title is still `32px`, whereas with this change, the height of the title is now `30px`, but the children are still `32px`. Reverting this change would fix the screenshots' "height" differences and still be in line with Kaleido.

Ooo thanks for catching that:

I fixed it here by removing the padding +1 calc instead:
adbf9c976d5aeae85ad0e19d89530840677a3346
caa9f83baf489869fc21d0a649b1d5a01b472039